docs/site/Creating-components.md

```diff
@@ -441,23 +441,35 @@ app.find('repositories.*');
 
 ## Configuring components
 
-More often than not, the component may want to offer different value providers
-depending on the configuration. For example, a component providing an email API
-may offer different transports (stub, SMTP, and so on).
+Components can be configured by an app by calling `this.configure()` in its
```

Comment:
CHANGE NOTE

Let's keep examples as simple as possible and avoid using concepts and terms which can be overwhelming for beginners.